cpp/src/arrow/c/bridge.cc

```diff
@@ -1418,9 +1454,7 @@ class ImportedBuffer : public Buffer {
 
 struct ArrayImporter {
   explicit ArrayImporter(const std::shared_ptr<DataType>& type)
-      : type_(type),
-        zero_size_buffer_(std::make_shared<Buffer>(kZeroSizeArea, 0)),
-        device_type_(DeviceAllocationType::kCPU) {}
```

Comment:
we lost the default to `DeviceAllocationType::kCPU`, should it be added to the initialization below?

Reply:
This was a linter action, which replaced DeviceAllocationType::kCPU with `{}` below since that's equivalent in this case. I'll replace `{}` with the more explicit form.

Reply:
That's an aggressive linter...

Reply:
i agree, that really is an aggressive linter.... lol